contract/test/test-build-proposal.js

```diff
@@ -4,10 +4,31 @@ import { execSync } from 'node:child_process';
 import { promises as fs } from 'node:fs';
 import { makeCompressFile } from './utils.js';
 
-test.before(t => (t.context.compressFile = makeCompressFile(fs.readFile)));
+/**
+ * @import {ExecutionContext} from 'ava';
+ */
+/**
+ * @typedef {object} TestContext
+ * @property {function(any): Promise<Buffer>} compressFile
+ */
+
+/**
+ * @param {ExecutionContext<TestContext>} t
+ */
+test.before(t => {
+  t.context = {
+    compressFile: makeCompressFile(fs.readFile),
+  };
+});
+
+/**
+ * @param {ExecutionContext<TestContext>} t
+ */
 
 test.only('proposal builder generates compressed bundles less than 1MB', async t => {
```

Comment:
drive-by, but please take `.only` out of the above test. It's already the only test in this module.